docs/reference/esql/functions/binary.asciidoc

```diff
@@ -3,14 +3,25 @@
 === Binary operators
 
 [[esql-binary-operators-equality]]
-==== Equality
+==== Equality `==`
 [.text-center]
 image::esql/functions/signature/equals.svg[Embedded,opts=inline]
 
 Supported types:
 
 include::types/equals.asciidoc[]
 
+==== Case Insensitive Equals `=~`
```

Comment:
Maybe `Case Insensitive Equality` is more consistent with the other operator names